apps/dialtone-documentation/docs/components/input-group.md

```diff
@@ -0,0 +1,294 @@
+---
+title: Input Group
+description: Input Groups are convenience components for a grouping of related inputs. While each input within the group could be independent, the v-model on the group provides a convenient interface for determining the current state of the group.
+status: ready
+thumb: true
+image: assets/images/components/input.png
+storybook: https://dialtone.dialpad.com/vue/?path=/story/components-input-group--default
+---
+
+<code-well-header>
+  <dt-input-group
+    name="fruits-input-group"
+    legend="Fruits"
+  >
+    <dt-radio value="apple"><span>Apple</span></dt-radio>
+    <dt-radio value="banana"><span>Banana</span></dt-radio>
+    <dt-radio value="other"><span>Other</span></dt-radio>
+  </dt-input-group>
+</code-well-header>
+
+## Default
+
+Input Groups are typically paired with a legend which identifies the group. If no legend is provided then it is expected that an `aria-label` will be given in order to provide an invisible label to screen readers. Each Input Group should contain one or more inputs which users can interact with.
+
+<code-well-header>
+  <dt-input-group
+    ref="baseExample"
+    name="fruits-input-group"
+  >
+    <dt-radio value="apple"><span>Apple</span></dt-radio>
+    <dt-radio value="banana"><span>Banana</span></dt-radio>
+    <dt-radio value="other"><span>Other</span></dt-radio>
+  </dt-input-group>
+</code-well-header>
+
+<code-example-tabs
+:htmlCode='() => $refs.baseExample'
+vueCode='
+<dt-input-group
+  name="fruits-input-group"
+>
+  <dt-radio value="apple"><span>Apple</span></dt-radio>
+  <dt-radio value="banana"><span>Banana</span></dt-radio>
+  <dt-radio value="other"><span>Other</span></dt-radio>
+</dt-input-group>
+'
+/>
+
+## Model
+
+The Vue model can have one of the following types `[String, Number, Boolean, Object]`:
+
+```js
+// default = null
+const value = 'some value';
+```
+
+The Vue model is dependant on the child component(s) implementing the provided `groupContext` and calling the provided `setGroupValue` method which will handle updating the provided `groupContext` and Vue model in the parent.
+
+```vue
+import {
+  DtInputMixin,
+  DtGroupableInputMixin,
+} from '@dialpad/dialtone-vue';
```

Comment:
We should import this from `@dialpad/dialtone/vue2` or `@dialpad/dialtone/vue3` instead of directly from `@dialpad/dialtone-vue` as maybe we deprecate the individual package.